PayPalNativeCheckout/src/main/java/com/paypal/android/checkout/PayPalClient.kt

```diff
@@ -23,7 +23,7 @@ class PayPalClient(application: Application, coreConfig: CoreConfig, returnUrl:
     init {
         val config = CheckoutConfig(
             application = application,
-            clientId = coreConfig.clientId,
+            clientId = clientId,
```

Comment:
+1 client ID can be removed here.

Reply:
`clientId` is requiered for PayPalNativeCheckout

Reply:
Got it yeah.